cartoframes/auth/credentials.py

```diff
@@ -52,9 +51,6 @@ def __init__(self, username=None, api_key='default_public', base_url=None, sessi
         if not is_valid_str(username) and not is_valid_str(base_url):
             raise ValueError('You must set at least a `username` or a `base_url` parameters')
 
-        if base_url is not None and urlparse(base_url).scheme != 'https':
```

Comment:
I would add a parameter like `allow_non_secure` (or similar) in the constructor, being `False` the default one

Reply:
Yes, I would add a param instead of removing this check. @cmongut could you evaluate the naming from the product point of view?

Reply:
👍 I'll add it. I was wondering about adding this parameter to Credentials `allow_non_secure` myself. opted for not adding it to avoid polluting the public api.